dashboard.html

```
@@ -0,0 +1,867 @@
+        <div class="command-center">
+            <div class="panel">
+                <div class="panel-header">
+                    <span class="panel-title">Command Center</span>
+                    <span style="font-family: var(--mono); font-size: 10px; color: var(--txt-muted);">paste JD or URL below · or pick a mode</span>
+                </div>
+                
+                <div class="jd-input-section">
+                    <p style="font-family: var(--mono); font-size: 11px; color: var(--txt-muted); margin-bottom: 8px;">/ career-ops — full auto-pipeline</p>
+                    <div class="jd-wrapper">
+                        <textarea id="jd-input" placeholder="Paste a job description or URL here... career-ops auto-detects and runs the full pipeline"></textarea>
+                        <button class="run-btn" onclick="runAutoPipeline()">Run ↗</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)